.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: node_js
+sudo: required
+dist: trusty
+node_js: stable
+addons:
+  firefox: latest
+  apt:
+    sources:
+      - google-chrome
+    packages:
+      - google-chrome-stable
+before_script:
+  - npm install -g bower polymer-cli@next --depth=0
+  - polymer install --variants
+cache:
+  directories:
+    - $HOME/.cache/yarn
+    - $HOME/.cache/bower
+    - $HOME/.npm
+    - bower_components-1.x
+    - bower_components
+script:
+  - xvfb-run polymer test -l chrome
```

Comment:
firefox wasn't playing nice so I put it in timeout. any help would be appreciated!